test/integration/targets/vmware_vmotion/tasks/main.yml

```diff
@@ -0,0 +1,79 @@
+# Test code for the vmware_vmotion module.
+# Copyright: (c) 2019, Pavan Bidkar <pbidkar@vmware.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- import_role:
+    name: prepare_vmware_tests
+  vars:
+    setup_attach_host: true
+    setup_virtualmachines: True
+    setup_datastore: true
+
+- name: Create VMs
+  vmware_guest:
+    hostname: "{{ vcenter_hostname }}"
+    username: "{{ vcenter_username }}"
+    password: "{{ vcenter_password }}"
```

Comment:
could you add:
```esxi_hostname: "{{ esxi1 }}"```

This way we are sure the VM is created on esxi1

Reply:
Done